examples/deployment/kubernetes/trillian-map-deployment.yaml

```diff
@@ -31,7 +31,7 @@ spec:
             command:
             - curl
             - -f
-            - http://localhost:8091/metrics
+            - http://localhost:8091/healthz
           failureThreshold: 3
           periodSeconds: 30
           timeoutSeconds: 30
```

Comment:
30 seconds is prob. too long given we only have 10? sec for live requests.

Reply:
done